README.rst

```diff
@@ -313,7 +310,7 @@ The ``version_id`` transport parameter enables you to get the desired version of
 
   >>> # If you don't specify a version, smart_open will read the most recent one
   >>> with open('s3://%s/%s' % (bucket, key)) as fin:
-  ...     print(repr(fin.read()))
+  ...     fin.read()
```

Comment:
Why this change? Intent less clear.

Reply:
Again, brevity. `print(repr(...))` is the default behavior of a REPL session.

Reply:
Our README goal is clarity, not brevity.

There is no assumption the code goes into REPL. Keep the `print` please.

Reply:
OK, reverted.